libraries/page/common/common.py

```diff
@@ -62,12 +62,12 @@ class footer:
     
 class pb:
     """PROFILE BUTTONS"""
-    myProfileBtn = '//button[text()="My Profile"]'
-    myBeansBtn = '//button[text()="My Beans"]'
-    wishlistBtn = '//button[text()="Wishlist"]'
-    myOrdersBtn = '//button[text()="My Orders"]'
-    mySubscriptionBtn = '//button[text()="My Subscriptions"]'
-    logOutBtn = '//button[text()=" Logout "]'
+    myProfileBtn = '//button[text()=" My Profile "]'
+    myBeansBtn = '//button[text()=" My Beans "]'
+    wishlistBtn = '//button[text()=" Wishlist "]'
+    myOrdersBtn = '//button[text()=" My Orders "]'
+    mySubscriptionBtn = '//button[text()=" My Subscriptions "]'
+    logOutBtn = '//button[text()="Logout"]'

```

Comment:
can we just contains(text(),) instead of text()= para if mabago ulit because of spaces, di na sia magmmatter.